wren-modeling-rs/core/tests/data/mdl.json

```diff
@@ -4,7 +4,7 @@
   "models": [
     {
       "name": "customer",
-      "tableReference": "main.customer",
+      "tableReference": "customer",
```

Comment:
Why remove the schema?

Reply:
The testing context, `RemoteContextProvider`, only register the table name without the schema. In this case, the table reference shouldn't have schema prefix.